tools/autograd/gen_python_functions.py

```diff
@@ -58,6 +58,31 @@
   ${unpack_self}
   ParsedArgs<${max_args}> parsed_args;
   auto r = parser.parse(args, kwargs, parsed_args);
```

Comment:
cc @albanD this may merge conflict with your stuff (hopefully not in a major way though)

Reply:
The warning stuff?

Reply:
Yep

Reply:
I'm not super familiar with this code... I don't see how they are linked?
What is your concern?